perun-base/src/test/resources/test-schema.sql

```diff
@@ -423,11 +410,11 @@ create table specific_user_users (
 	modified_by_uid integer,
 	modified_at timestamp default current_date not null,
 	type longvarchar default 'service' not null,
-	status char(1) default '0' not null, --is it service user?
+	status integer default 0 not null, --is it service user?
```

Comment:
This comment is misleading. Status in this table means 'enabled or disabled ownership'. Can you please change it here?

Reply:
Fixed in both DB definition files.